src/applications/RunMachineShop.java

```diff
@@ -0,0 +1,70 @@
+package applications;
+
+public class RunMachineShop {
+
+    /** process all jobs to completion
+     * @param simulationResults*/
+    static void simulate(SimulationResults simulationResults) {
+        while (JobManager.numJobs > 0) {// at least one job left
+            int nextToFinish = MachineShopSimulator.eList.nextEventMachine();
+            MachineShopSimulator.timeNow = MachineShopSimulator.eList.nextEventTime(nextToFinish);
```

Comment:
These 2 lines point to me that either these fields belong here in RunMachineShop, or that MachineShopSimulator and RunMachineShop have inappropriate intimacy/feature envy.